.ci/metrics/metrics.py

```diff
@@ -191,25 +196,23 @@ def get_per_workflow_metrics(
 
             # The timestamp associated with the event is expected by Grafana to be
             # in nanoseconds.
-            created_at_ns = int(created_at.timestamp()) * 10**9
+            completed_at_ns = int(completed_at.timestamp()) * 10**9
```

Comment:
This should be better than what we're doing now, but isn't guaranteed to not try and upload historical information. I think going by the workflow completion time should fix that, but I couldn't a good way to get that out of the workflow, other than just iterating through all the jobs.

Reply:
Yep, added a condition to check the metric age before adding it, thus dropping metrics older than 2 hours.